lib/agris/api/grain/commodity_codes.rb

```diff
@@ -3,7 +3,7 @@ module Agris
   module Api
     module Grain
       module CommodityCodes
-        def commodity_code(location, code)
+        def commodity_code(location = '', code = '')
```

Comment:
Wouldn't an endpoint named `commodity_codes` be more in the spirit of the existing client methods?

Reply:
this method calls commodity_codes already or I would have

Reply:
@josephbridgwaterrowe 

I don't see anywhere we actually intend to receive multiple records right now. But all of the Agris endpoints do return multiple.

The Agris Documentation name for this endpoint is Commodity Code. I think so far we've just been naming them off of the endpoint name.

Reply:
Ahh... so these changes don't actually do anything?

A user would use the commodity_codes method to fetch more?